code/iaas/api-logic/src/main/java/io/cattle/platform/iaas/api/auth/dao/impl/AuthDaoImpl.java

```diff
@@ -47,8 +47,7 @@ public Account getAccountById(Long id) {
                 .selectFrom(ACCOUNT)
                 .where(
                         ACCOUNT.ID.eq(id)
-                        .and(ACCOUNT.STATE.eq("active")
-                        .and(ACCOUNT.REMOVED.isNull()))
+                        .and(ACCOUNT.STATE.ne("purged"))
```

Comment:
`io.cattle.platform.iaas.api.auth.impl.GithubOAuthImpl.getAccount(ApiRequest)` uses this method.  I think this may mean that non-active projects will be authenticated.